demos/_assets/css/jqm-demos.css

```diff
@@ -1000,7 +1013,8 @@ li.jqm-3rd-party .ui-listview-item-button::before {
 }
 
 .jqm-search-results-highlight {
-	color: #0e82a5;
+	background: #77D1EF;
```

Comment:
Lowercase letters in color code (http://contribute.jquery.org/style-guide/css/#formatting)